docs/install.rst

```diff
@@ -470,20 +476,12 @@ can also open issues in the repositories for some of the dependencies:
 Testing a Source Code Build of ``astropy``
 ------------------------------------------
 
-Before running tests, it is necessary to make sure that Astropy's test
-dependencies are installed. This can be done with the following command::
-
-    pip install pytest-astropy
-
-More information on what the ``pytest-astropy`` package provides can be found
-in :ref:`testing-dependencies`.
-
-The most convenient way to test that your Astropy built correctly
-(without installing ``astropy``) is to run this from the root of the source tree::
+The easiest way to run the tests in a source checkout of ``astropy``
+is to use `tox <https://tox.readthedocs.io/en/latest/>`_::
 
-    python setup.py test
+    tox -e test-all
```

Comment:
should this be `test-alldeps` (or `test-all-dependencies`)?

Reply:
Fixed